Paws4Me/Paws4Me/Extensions/UIImageExtension.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  UIImageExtension.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/09.
+//
+
+import Foundation
+import UIKit
+
+extension UIImage {
+    public static func loadFrom(url: URL, completion: @escaping (_ image: UIImage?) -> Void) {
```

Comment:
So all these functions can just be normal functions 😄 no need for `static`

Reply:
Thank you for this. I actually need it tp be static as the function right underneath it uses this function loadFrom (line 28)

Reply:
Thank you👍🏻

Reply:
So the thought process here is you don't want to call these functions as static functions: 
`UIImage.loadFrom....`

You want to use it like this:
`imageView.loadFrom(url: "")` 

Handle all the logic in the `extension` otherwise you'll still be duplicating code when you call this function 😄 drop me a message on slack if you want to chat about this?